### ISSUE TITLE
Need for for feeder/provider template?

### ISSUE THREAD

**Issue by erikbosch** [eclipse/kuksa.val.feeders#162]
I noticed #161 and started to think how we want to handle new contributions in the future when we have separate repositories for each feeder/provider/service. Then it is not sufficient like  in #161 to just add a few files, you need to add additional files (LICENSE, .gitignore, CI-files, ...). I am thinking if it would make sense to have a dedicated minimalist repo that could be used as designbase, possibly called `kuksa-provider-template`, `kuksa-example-provider` or `kuksa-repo-template`. It could even be a [Github template repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository)

**Comment by lukasmittag** [comment]
Yes, like in eclipse velocitas maybe. Sounds definetly good. CI wise also very good approach I guess. But we shouldn't forget to do a documentation somewhere and state it on the front page. Then the "user" does not need to search this long and we have better PRs without redirecting/explaining them.

**Comment by lukasmittag** [comment]
We could even add a devcontainer again. In it just Databroker and/or some example provider (dbc-provider?)

**Comment by erikbosch** [comment]
Maybe we should add a README to our (new) Github root, like for [https://github.com/eclipse-velocitas](https://github.com/eclipse-velocitas). Compare with [https://github.com/eclipse-kuksa](https://github.com/eclipse-kuksa) which is quite empty right now.

If we have a README there some text on how to propose/contribute new repos make sense. Like creating a new repo using the template and then create an issue in `kuksa-common` (if we want to use that repo for every request that does not fit elsewhere)


[How to add organization README](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme)
